### PR TITLE
add documentation on method withoutAnyArgument

### PR DESCRIPTION
**Pull request by agallou**

### CODE REVIEW

File: source/fr/assertions/mock.inc.rst [hunk 1]
```diff
@@ -260,8 +260,34 @@ withAtLeastIdenticalArguments
 .. warning::
    | ``withAtLeastIdenticalArguments`` teste le type des arguments.
    | Si vous ne souhaitez pas vérifier leurs types, utilisez :ref:`withAtLeastArguments <with-at-least-arguments>`.
+u
```
Comment (marmotz) [inline source/fr/assertions/mock.inc.rst]: ???

Comment (agallou) [inline source/fr/assertions/mock.inc.rst]: fixed

**Comment by marmotz** [comment]
Thank you for all your work !

**Comment by agallou** [comment]
you're welcome.

by the way, @marmotz have you seen https://github.com/atoum/stubs/pull/4/files ? it integrates the documentation into the stubs.

**Comment by marmotz** [comment]
Manually ? Automatically ?
We have to maintain "2 documentations" (doc and stub) ?

**Comment by agallou** [comment]
with a script that updates the phpdoc from the text output of the documentation.

**Comment by marmotz** [comment]
ok

**Pull request merged**